frontend/test/metabase/scenarios/dashboard/reproductions/27356-swtiching-between-dashboards.cy.spec.js

```diff
@@ -0,0 +1,73 @@
+import {
+  restore,
+  openNavigationSidebar,
+  navigationSidebar,
+} from "__support__/e2e/helpers";
+import { SAMPLE_DATABASE } from "__support__/e2e/cypress_sample_database";
+
+const { REVIEWS_ID } = SAMPLE_DATABASE;
+describe("issue 27356: 'Something's gone wrong' error when moving between dashboards that has/hasn't filters", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  it("should not error when navigating from a dashboard with filters to a dashboard without filters", () => {
+    cy.intercept("GET", "/api/dashboard/*").as("getDashboard");
+    const questionDetails = {
+      name: "27356",
+      query: { "source-table": REVIEWS_ID },
+    };
+
+    const ratingFilter = {
+      name: "Rating Filter",
+      slug: "rating",
+      id: "5dfco74e",
+      type: "string/=",
+      sectionId: "string",
+    };
+
+    cy.createQuestionAndDashboard({
+      questionDetails,
+      dashboardDetails: {
+        parameters: [ratingFilter],
+        name: "Test With Params",
+      },
+    }).then(({ body: { dashboard_id: dashboardWithFilter } }) => {
+      cy.createQuestionAndDashboard({
+        questionDetails,
+        dashboardDetails: { name: "Test Without Params" },
+      }).then(async ({ body: { dashboard_id: dashboardWithOutFilter } }) => {
+        await cy.request(
+          "POST",
+          `/api/bookmark/dashboard/${dashboardWithFilter}`,
+        );
+        await cy.request(
+          "POST",
+          `/api/bookmark/dashboard/${dashboardWithOutFilter}`,
+        );
```

Comment:
Cypress doesn't work with `async/await`. Unfortunately, this all requires further chaining with `.then` (callback hell). Let me see if there's an easier way to repro.
```suggestion
      }).then(({ body: { dashboard_id: dashboardWithOutFilter } }) => {
        cy.request(
          "POST",
          `/api/bookmark/dashboard/${dashboardWithFilter}`,
        );
        cy.request(
          "POST",
          `/api/bookmark/dashboard/${dashboardWithOutFilter}`,
        );
```